packaging/createpkg.py

```diff
@@ -85,7 +85,7 @@ def generate_distribution(self):
         ET.SubElement(root, 'pkg-ref', {'id': self.identifier})
         ET.SubElement(root, 'options', {'customize': 'never',
                                         'require-scripts': 'false',
-                                        'hostArhcitectures': 'x86_64,arm64'})
+                                        'hostArchitectures': 'x86_64,arm64'})
```

Comment:
scary?

Reply:
Probably this would be a functional change for the macOS packages, although I don't know how many people actually use those.

Reply:
I installed `meson` via `brew install meson` when i was trying to figure out the `zsh` thing... I'm not sure if that counts.

Reply:
It doesn't, brew uses pip basically. createpkg.py creates something that could be added to the macOS App Store or whatever.